inc/options/cdn.php

```diff
@@ -186,173 +367,386 @@
+								// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag.
+								__(
+									'Specify user agents that should not access files hosted with the %1$sCDN%2$s.',
+									'w3-total-cache'
+								),
+								'<acronym title="' . __( 'Content Delivery Network', 'w3-total-cache' ) . '">',
+								'</acronym>'
+							),
+							array(
+								'acronym' => array(
+									'title' => array(),
+								),
+							)
+						);
+						?>
+					</p>
 				</td>
 			</tr>
 			<tr>
-				<th><label for="cdn_reject_files"><?php Util_Ui::e_config_label( 'cdn.reject.files' ) ?></label></th>
+				<th><label for="cdn_reject_files"><?php Util_Ui::e_config_label( 'cdn.reject.files' ); ?></label></th>
 				<td>
 					<textarea id="cdn_reject_files" name="cdn__reject__files"
-						<?php Util_Ui::sealing_disabled( 'cdn.' ) ?> cols="40" rows="5"><?php echo esc_textarea( implode( "\r\n", $this->_config->get_array( 'cdn.reject.files' ) ) ); ?></textarea>
-					<p class="description"><?php _e( 'Specify the path of files that should not use the <acronym title="Content Delivery Network">CDN</acronym>.', 'w3-total-cache' ) ?></p>
+						<?php Util_Ui::sealing_disabled( 'cdn.' ); ?> cols="40" rows="5"><?php echo esc_textarea( implode( "\r\n", $this->_config->get_array( 'cdn.reject.files' ) ) ); ?></textarea>
+					<p class="description">
+						<?php
+						echo wp_kses(
+							sprintf(
+								// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag.
+								__(
+									'Specify the path of files that should not use the %1$sCDN%2$s.',
+									'w3-total-cache'
+								),
+								'<acronym title="' . __( 'Content Delivery Network', 'w3-total-cache' ) . '">',
+								'</acronym>'
+							),
+							array(
+								'acronym' => array(
+									'title' => array(),
+								),
+							)
+						);
+						?>
+					</p>
 				</td>
 			</tr>
 			<tr>
 				<th colspan="2">
 					<input type="hidden" name="set_cookie_domain_old" value="<?php echo (int) $set_cookie_domain; ?>" />
 					<input type="hidden" name="set_cookie_domain_new" value="0" />
 					<label><input type="checkbox" name="set_cookie_domain_new"
-						<?php Util_Ui::sealing_disabled( 'cdn.' ) ?> value="1"<?php checked( $set_cookie_domain, true ); ?> /> <?php printf( __( 'Set cookie domain to &quot;%s&quot', 'w3-tota-cachel' ),  htmlspecialchars( $cookie_domain ) ) ?></label>
-					<p class="description"><?php _e( 'If using subdomain for <acronym title="Content Delivery Network">CDN</acronym> functionality, this setting helps prevent new users from sending cookies in requests to the <acronym title="Content Delivery Network">CDN</acronym> subdomain.', 'w3-total-cache' ) ?></p>
+						<?php Util_Ui::sealing_disabled( 'cdn.' ); ?> value="1"<?php checked( $set_cookie_domain, true ); ?> />
+						<?php
+						echo esc_html(
+							sprintf(
+								// translators: 1 Cookie Domain.
+								__(
+									'Set cookie domain to "%1$s"',
+									'w3-tota-cachel'
```

Comment:
Typo.  Fix the text-domain to "w3-total-cache".

Reply:
Fixed typo